<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotationEditor</class>
 <widget class="QWidget" name="RotationEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="m_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QLabel" name="m_label_x">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="m_x">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>23</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="m_label_y">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="m_y">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>49</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="m_label_z">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="m_z">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>75</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-90.000000000000000</double>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="m_slider_x">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>23</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="m_slider_y">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>49</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="m_slider_z">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>75</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
